README.md

```diff
@@ -1,304 +1,125 @@
+* `To Mount`
+  - Launch the provided GUI escalated with admin rights (this is for the windows service option).
+  - To attach your cloud to a folder, browse to the location you want your cloud to mount and make sure that directory is empty.
+  - Select the type of cloud you want, either S3 or Azure.
+  - Click mount.
+  - You will see an error if you did not launch the GUI with admin rights, if this happens just relaunch the GUI by right-clicking and selecting 'Run as Administrator'.
+
+* `To Unmount`
+  - Launch the provided GUI with admin rights.
+  - Through the main window of the GUI, browse to the cloud you want unmounted and select that folder
+  - Select the type of cloud you have mounted, either S3 or Azure.
+  - Click unmount.
+
+If you do not desire to use the GUI to mount/unmount your cloud, please refer to Command Line Interface Operations below.
 
-See [here](WINDOWS.md) for how to setup Cloudfuse to run on Windows.
+## Health Monitor
+Cloudfuse also supports a health monitor. It allows customers gain more insight
+into how their Cloudfuse instance is behaving with the rest of their machine.
+Visit [here](https://github.com/Seagate/cloudfuse/wiki/Health-Monitor) to set it up.
+
+## Command Line Interface Operations
 
-## Supported Operations
-The general format of the Cloudfuse commands is `cloudfuse [command] [arguments]
+### Linux Commands:
+The general format of the Cloudfuse Linux commands is `cloudfuse [command] [arguments]
 --[flag-name]=[flag-value]`
 * `help` - Help about any command
 * `mount` - Mounts a cloud storage container as a filesystem. The supported
-  containers include
-  - S3 Bucket
-  - Azure Blob Container
-  - Azure Datalake Gen2 Container
-* `mount all` - Mounts all the containers in an S3 Account or Azure account as a
-  filesystem. The supported storage services include
-  - [S3 Storage](https://aws.amazon.com/s3/)
-  - [Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
-  - [Datalake Storage Gen2](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction)
+  containers include:
+  - [S3 Bucket Storage](https://aws.amazon.com/s3/)
+  - [Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+  - [Azure Datalake Storage Gen2](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction)
+  - Example: cloudfuse mount \<mount path> --config-file=\<config file>
+* `mount all` - Mounts all the containers in an S3 Account or Azure account supported by mount
+  - Example: cloudfuse mount all \<mount path> --config-file=\<config file>
 * `mount list` - Lists all Cloudfuse filesystems.
+  - cloudfuse mount list
+* `unmount` - Unmounts the Cloudfuse filesystem.
+  - Example: cloudfuse unmount \<mount path>
+* `unmount all` - Unmounts all Cloudfuse filesystems.
+  - Example: cloudfuse unmount all
+* `secure decrypt` - Decrypts a config file.

```

Comment:
These `secure` options are already described below, so we could drop them here.

Reply:
I will add another sub-heading to make it obvious both OS's can use them